src/api/index.ts

```diff
@@ -1,27 +1,28 @@
 import {AptosClient, Types} from "aptos";
+import {isHex} from "../pages/utils";
 import {withResponseError} from "./client";
 
 export async function getTransactions(
-  requestParameters: {start?: number; limit?: number},
+  requestParameters: {start?: bigint; limit?: number},
   nodeUrl: string,
-): Promise<Types.OnChainTransaction[]> {
+): Promise<Types.Transaction[]> {
   const client = new AptosClient(nodeUrl);
   const transactions = await withResponseError(
     client.getTransactions(requestParameters),
   );
 
   // Sort in descending order
   transactions.sort((a, b) =>
-    parseInt(a.version) < parseInt(b.version) ? 1 : -1,
+    parseInt((a as any).version) < parseInt((b as any).version) ? 1 : -1,
```

Comment:
Please do not cast values to `any`: it disables type checking and eventually leads to runtime errors. A better option here would be to narrow the type (e.g. with `'version' in a`). It's often helpful to hover over values in your editor to see what the inferred type is.